.github/workflows/release.yml

```diff
@@ -60,6 +68,9 @@ jobs:
           fetch-depth: 0
```

Comment:
Please add a documentation hint for what this is good for.

Reply:
Fixed